lib/v2/screens/profile_screens/security/components/security_card.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/v2/components/divider_jungle.dart';
+import 'package:seeds/v2/domain-shared/ui_constants.dart';
+import 'package:seeds/design/app_theme.dart';
+
+/// SECURITY CARD
+class SecurityCard extends StatelessWidget {
```

Comment:
also, the ripple effect is not 100% correct, the corners get clipped. 
You are missing Ink on your Inkwell

This is also happening in the cards in Settings. Try to make generic and reuse between screens.

Reply:
Got it I'll fix the ripple